packages/devextreme/js/__internal/ui/context_menu/m_context_menu.ts

```diff
@@ -531,6 +531,7 @@ class ContextMenu extends MenuBase {
 
   _renderSubmenuItems(node, $itemFrame: dxElementWrapper): void {
     this._renderItems(this._getChildNodes(node), $itemFrame);
+    this._planPostRenderActions();
```

Comment:
I see that this function is called on render in `async` strategy, so not it's called twice? Doesn't it have side affects?
It's more expected for me if we call it once after everything is rendered

Reply:
unfortunately, submenu is not contextMenu instance, and this method not called for it, so we have to call it manually after all submenu items are rendered